wrench/src/angle.rs

```diff
@@ -0,0 +1,68 @@
+use glutin;
+use glutin::{WindowBuilder, ContextBuilder, EventsLoop, Window, CreationError};
+
+#[cfg(not(windows))]
+pub enum Context {}
+
+#[cfg(windows)]
+pub use ::egl::Context;
+
+impl Context {
+    #[cfg(not(windows))]
+    pub fn with_window(
+        _: WindowBuilder,
+        _: ContextBuilder,
+        _: &EventsLoop,
+    ) -> Result<(Window, Self), CreationError> {
+        Err(CreationError::PlatformSpecific("ANGLE rendering is only supported on Windows".into()))
+    }
+
+    #[cfg(windows)]
+    pub fn with_window(
+        window_builder: WindowBuilder,
+        context_builder: ContextBuilder,
+        events_loop: &EventsLoop,
+    ) -> Result<(Window, Self), CreationError> {
+        use glutin::os::windows::WindowExt;
+
+        // FIXME: &context_builder.pf_reqs  https://github.com/tomaka/glutin/pull/1002
+        let pf_reqs = &glutin::PixelFormatRequirements::default();
```

Comment:
At the moment Wrench does not call any `ContextBuilder` method that touches the pixel format requirements, so using `default()` doesn’t make a difference.